react/more_react_concepts/refs_and_memoization.md

```diff
@@ -300,7 +300,7 @@ Phew, this was a long lesson. Refs and memoization are difficult concepts to gra
 <div class="lesson-content__panel" markdown="1">
 
 1. The article [When to useMemo and useCallback](https://kentcdodds.com/blog/usememo-and-usecallback) by Kent C. Dodds further introduces more examples of when to use `useMemo` and `useCallback` and when you shouldn't bother using them.
-1. We've only learned about a basic implementation of the `useRef` hook. For more examples about its usage and why we should be wary of using the hook (more on the links they provided in the guide), check out the [interactive guide of the React documentation](https://react.dev/reference/react/useRef) for `useRef`.
+1. We've only learned about a basic implementation of the `useRef` hook. For more examples about its usage and why we should be wary of using the hook (more on the links they provided in the guide), check out the interactive guide of the React documentation for [`useRef` hook](https://react.dev/reference/react/useRef) .
```

Comment:
```suggestion
1. We've only learned about a basic implementation of the `useRef` hook. For more examples about its usage and why we should be wary of using the hook (more on the links they provided in the guide), check out the interactive guide of the React documentation for [useRef hook](https://react.dev/reference/react/useRef) .
```
Forgot to add this to the review as well. Good spot on this since I somehow missed adding it to the spreadsheet.
Your intuition is correct, in this case (2 words, 1 with backticks) the single backticked word makes the link harder to spot/looks like 2 separate links. So in this scenario, just not using backticks looks perfectly fine. With more non-backticked words in the link text, this becomes less of a visual issue.